### PR TITLE
DAG-1669: Specify when generated tasks should be run on large distros

### PR DESCRIPTION
**Pull request by dbradf**

### CODE REVIEW

File: src/evergreen_names.rs [hunk 1]
```diff
@@ -63,3 +65,7 @@ pub const TASK_NAME: &str = "task";
 pub const IDLE_TIMEOUT: &str = "timeout_secs";
 /// Multiversion version combination being run against.
 pub const MULTIVERSION_EXCLUDE_TAGS: &str = "multiversion_exclude_tags_version";
+
+// Build Variant expansions.
+/// Name of large distro for build variant.
+pub const LARGE_DISTRO_NAME: &str = "large_distro_name";
```
Comment (zamj) [inline src/evergreen_names.rs:71]: Can you change this constant name to something like `LARGE_DISTRO_EXPANSION_NAME` or `LARGE_DISTRO_EXPANSION`? The current name confused me for a second and I thought this was the name of the distro in Evergreen.

**Review by zamj — APPROVED** [review]
LGTM, left one comment

File: src/evergreen_names.rs (hunk 1 above)
Comment (dbradf) [reply src/evergreen_names.rs]: done.

**Comment by dbradf** [comment]
evergreen merge

**Pull request merged**